### ISSUE TITLE
Premier coup d'oeil

### ISSUE THREAD

**Issue by jhb2** [vibby/cuisine-facile#1]
Wouh, ça commence bien...
Bon, juste 2 remarques au passage
- je suis pas super fan de la couleur rouge pour les menus
- le titre de la recette ne serait-il pas mieux dans une autre police que le titre du site (plus petite peut-être ? ou l'inverse ?)
  C'est vraiment un beau début en tout cas !

**Comment by vibby** [comment]
Pour les couleurs, tu peux faire des propositions ? Perso je m’aide de cet outil : https://color.adobe.com/Flat-UI-color-theme-2469224/edit/?copy=true
On y trouve plein de propositions de couleurs qui marchent ensemble : https://color.adobe.com/explore/most-popular/?time=all
À mon avis, ce serais mieux de garder des bleus pour garder une parenté avec le site existant.

Pour le titre de la recette, c’est un truc vite fait pour éviter le contenu vide.

**Comment by jhb2** [comment]
Je me méfie de mes propres propositions j'ai pas des gouts très surs... Mais bon je suis juste pas un fan du bleu + rouge. Bref, je te fais entièrement confiance :-)
Je suis d'accord avec toi pour le bleu, c'est un peu la signature de l'ancien site et c'est bien de garder une continuité.
a+

**Comment by vibby** [comment]
J’ai remplacé le rouge par un orange.
J’ai commencé a travailler la zone de recette, il me reste toute la partie «informations».

**Comment by jhb2** [comment]
ça commence a avoir de la gueule !
C'est voulu que le déroulé de la recette soit à gauche de la description en "grand écran" et passe sous si on réduit la taille de l'écran ?

**Comment by vibby** [comment]
Oui, l’idée du responsive, ce n’est pas seulement d’afficher mieux sur mobile, mais aussi de mieux profiter des grands écran.
